admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -32,17 +32,19 @@ of memcache that best fits your needs. The supported caching backends are:
    A local cache for systems running PHP 5.5 and up, APCu 4.06 or better required.
 * `Memcached <http://www.memcached.org/>`_ 
    Distributed cache for multi-server ownCloud installations.
-* `Redis <http://redis.io/>`_ 
+* `Redis <http://redis.io/>`_, PHP module 2.2.5 or better required.
    For distributed caching, and required for Transactional File Locking.
 
 These are supported but not recommended:
 
-* `XCache <http://xcache.lighttpd.net/>`_ 
+* `XCache <http://xcache.lighttpd.net/>`_ (might need some special configuration
+  not described in this documentation)
```

Comment:
I'm also fine with that. Had used XCache together with PHP 5.4 and PHP 5.5 on oC 7 (and 8?) without any problems but havn't tested PHP 5.6 with oC 8.1.